backend/src/utils/config.ts

```diff
@@ -187,4 +190,5 @@ export interface Config {
 }
 
 const config: Config = _config.util.toObject()
+config.ui.tokenActions = TokenDescriptions
```

Comment:
I think this is a clear move away from how the UI config is used and having things like this where the backend appends to the configuration that is provided by the configuration file could easily add confusion in the future. I think having to do this shows that this token information isn't suitable for the UI config. The UI config contains settings that can differ per deployment and need to be shared between the frontend and the backend. The token available actions and their descriptions fulfil the latter but not the former. Instead I'd suggest a simple token endpoint that returns an object of Bailo's available token actions with their descriptions so clients like the UI can display that information to users